### DISCUSSION TITLE
How to connect to redis cluster

### DISCUSSION THREAD

**Discussion by liuqijie6** [miguelgrinberg/Flask-SocketIO#1965] [Q&A]
socket_io.init_app(app,
                       message_queue=f'redis://:{r_base.password}@{r_base.host}:{r_base.port}/0',
                       )
I have nodes = [xx1,xx2,xx3,xx4,xx5,xx6]

How to connect to redis cluster ?

**Comment by miguelgrinberg** [comment]
Cluster connections are not currently implemented. This would need to be added in [this class](https://github.com/miguelgrinberg/python-socketio/blob/main/src/socketio/redis_manager.py).

**Reply by liuqijie6** [reply]
emmmm  Can you be more specific? I didn't understand, thank you

**Reply by miguelgrinberg** [reply]
I can't be more specific. Clusters cannot currently be used, as I said above.

**Reply by liuqijie6** [reply]
Hey can you update the support for redis cluster connections? if you are free

**Reply by miguelgrinberg** [reply]
This isn't a priority right now, but I'll add it to my list of tasks for the future.
